.github/workflows/push-to-prod.yml

```diff
@@ -78,3 +78,27 @@ jobs:
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN}}
           balena_command: "deploy nebraltd/helium-indoor-470 --logs"
+
+  open-fleet:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Balena Deploy
+        uses: nebraltd/balena-cli-action@v12.48.12
+        if: success()
+        with:
+          balena_api_token: ${{secrets.BALENA_API_TOKEN}}
+          balena_command: "deploy nebraltd/helium-miner --logs"
```

Comment:
What are these new fleets going to be used for?

Reply:
@vpetersson and I were talking yesterday about a way to eventually move to just two different fleets:
- helium-miner which would be the main one for non-paying customers using balena open fleets
- helium-miner-commercial which would be for paying customers using our SaaS platform

This was an initial commit to allow us to begin testing those in the future (there is a fair bit of other work that would need to be done to enable this fully)